src/lib/openapi/spec/create-user-response-schema.ts

```diff
@@ -0,0 +1,34 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { userSchema } from './user-schema';
+
+export const createUserResponseSchema = {
+    $id: '#/components/schemas/createUserResponseSchema',
+    type: 'object',
+    additionalProperties: false,
+    description: 'An Unleash user after creation',
+    required: ['id'],
+    properties: {
+        ...userSchema.properties,
```

Comment:
same as user schema but it returns either number or string for the role name depending on what was in the create/update request

Reply:
user schema hasn't changed because this schema was added